src/applications/static-pages/translation.js

```diff
@@ -10,17 +10,24 @@ const faqI18Content = {
     langToggleLink: '/coronavirus-veteran-frequently-asked-questions-esp',
   },
 };
-
+const configureBreadCrumbLinks = currentLang => {
+  const breadCrumbLinks = document.getElementsByClassName('breadcrumb-link');
+  for (let i = 0; i < breadCrumbLinks.length; i++) {
```

Comment:
Might want to add something like:

```js
if (!breadCrumbLinks) {
  return;
}
```

just in case we're not able to find them and it throws a runtime error on `breadCrumbLinks.length` 🙂